contrib/test/integration/vars.yml

```diff
@@ -3,6 +3,24 @@
 # When swap setup is necessary, make it this size
 swapfileGB: 8
 
+# Path to encode into /etc/environment on all hosts
+go_path: "/go"
+
+# Absolute path on control-host where the cri-o source exists
+cri_o_src_path: "{{ playbook_dir }}/../../../"
+
+# Absolute path on subjects where cri-o source is expected
+cri_o_dest_path: "{{ go_path }}/src/github.com/kubernetes-incubator/cri-o"
+
+# Global environment variables to use for all tasks
+environment_variables:
+    PATH: "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:{{ go_path }}/bin:/usr/local/go/bin"
+    GOPATH: "{{ go_path }}"
+    KUBERNETES_PROVIDER: "local"
+    KUBECONFIG: "/var/run/kubernetes/admin.kubeconfig"
+    CGROUP_MANAGER: "cgroupfs"
+    STORAGE_OPTS: '--storage-driver=overlay{{ extra_storage_opts | default("") }}'
```

Comment:
Also, in my opinion it's much harder to reason about how `extra_storage_opts` gets set in the integration playbook, but then used in a round-a-bout way to create this env var to pass to the `make` target...

Reply:
Totally agree, I just fixed that as a tangent to one of Ed's suggestions.  Now I see here, you must have been reading my mind...from the future 😀